client/my-sites/marketing/buttons/preview-buttons.jsx

```diff
@@ -2,14 +2,18 @@ import classNames from 'classnames';
 import { localize } from 'i18n-calypso';
 import { filter, isEqual } from 'lodash';
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-import ReactDom from 'react-dom';
+import { Component, createRef } from 'react';
 import ResizableIframe from 'calypso/components/resizable-iframe';
 import { hasTouch } from 'calypso/lib/touch-detect';
 import ButtonsPreviewButton from 'calypso/my-sites/marketing/buttons/preview-button';
 import previewWidget from './preview-widget';
 
 class SharingButtonsPreviewButtons extends Component {
+	constructor( props ) {
+		super( props );
+		this.previewIframe = createRef();
```

Comment:
Need to declare `this.moreButton`, too, otherwise the `this.moreButton.current` statements will be crashing.